schemas/context/experienceevent.schema.json

```diff
@@ -18,7 +18,13 @@
   "description": "An ExperienceEvent captures observations, including the point in time and identity of the subject involved. Experience Events are fact records of what occurred, representing what happened without aggregation or interpretation. They are critical for time-domain analytics as they allow for observation and analysis of changes that occur in a given window of time and the comparison between multiple windows of time to track trends. Experience Events can be either explicit (directly observable human actions) or implicit (raised without a direct human action).",
   "definitions": {
     "experienceevent": {
-      "properties": { }
+      "properties": {
+        "xdm:correlationID": {
```

Comment:
This should be named xdm:corrleationId (lower case d) to match the naming format of other ID fields. 

Also, I'm assuming this field is always populated by the data producer before Platform ingestion and not something that happens internally by the Platform so we may want to specify that in the description.

Reply:
@cmathis Is this naming convention enforced right now? I ask this because there are quite a few inconsistencies in the base schemas:
* [context/device](https://github.com/adobe/xdm/blob/master/docs/reference/context/device.schema.md): `xdm:typeID` and `xdm:typeIDService`
* [context/search](https://github.com/adobe/xdm/blob/master/docs/reference/context/search.schema.md): `xdm:searchEngineID`
* [context/segmentMembershipItem](https://github.com/adobe/xdm/blob/master/docs/reference/context/segmentmembershipitem.schema.md): `xdm:profileStitchID`, `xdm:segmentID`
* [assets/asset](https://github.com/adobe/xdm/blob/master/docs/reference/assets/asset.schema.md): `xdm:documentID`, `xdm:aliasIDs`
* [data/order](https://github.com/adobe/xdm/blob/master/docs/reference/data/order.schema.md): `xdm:purchaseID`

To sum up, I'd like to understand:
1. Is the `Id` (lower case "d") suffix preferred going forward? Is it also enforced?
2. Is there a plan to make base schemas consistent with this naming convention?

Reply:
The style guide uses `ID` so this field is named correctly.